.github/workflows/release-build-all.yml

```diff
@@ -73,7 +74,7 @@ jobs:
           ref: ${{ needs.createTag.outputs.newTag }}
       - name: Log in to OpenShift
         run: |
-          oc login --token=${{ secrets.SA_TOKEN }} --server=https://api.silver.devops.gov.bc.ca:6443
+          oc login --token=${{ secrets.SA_TOKEN }} --server=${{ vars.OPENSHIFT_CLUSTER_URL }}
```

Comment:
One advantage of having those vars is that if the value changes it is only one place to be changed.